Source/WebCore/html/canvas/CanvasRenderingContext2DBase.h

```diff
@@ -362,12 +335,41 @@ class CanvasRenderingContext2DBase : public CanvasRenderingContext, public Canva
             DidDrawOption::ApplyClip,
         };
     }
-
     void didDraw(std::optional<FloatRect>, OptionSet<DidDrawOption> = defaultDidDrawOptions());
     void didDrawEntireCanvas(OptionSet<DidDrawOption> options = defaultDidDrawOptions());
     void didDraw(bool entireCanvas, const FloatRect&, OptionSet<DidDrawOption> options = defaultDidDrawOptions());
     template<typename RectProvider> void didDraw(bool entireCanvas, RectProvider, OptionSet<DidDrawOption> options = defaultDidDrawOptions());
 
+    static String normalizeSpaces(const String&);
```

Comment:
These seem to move but they're just because didDraw and needed things have to move to protected, since there are derived classes that draw.